app/models/project.rb

```diff
@@ -1,3 +1,14 @@
 class Project < ApplicationRecord
   belongs_to :challenge
+  has_many :contestant_projects
+  has_many :contestants, through: :contestant_projects
+
+  def contestants_avg_xp
+    total_experience = contestants.sum(:years_of_experience)
```

Comment:
This implementation is even easier! Instead of doing `sum` here, just try `average`!

contestants.average(:years_of_experience)

that's it!

If you're ever doing a bunch of Ruby logic in a model method, try to stop yourself and consider: is there another ActiveRecord method that could make this even easier? Usually there is!